src/webgpu/shader/execution/flow_control/harness.ts

```diff
@@ -217,6 +217,13 @@ ${main_wgsl.extra}
         // returns an Error with the given message and WGSL source
         const fail = (err: string) => Error(`${err}\nWGSL:\n${Colors.dim(Colors.blue(wgsl))}`);
 
+        // returns a string that shows the outputted values to help understand the whole trace.
+        const print_output_value = () => {
+          return `Output values (length: ${outputCount}): ${outputs.data
+            .slice(1, outputCount + 1)
```

Comment:
nit: drop the `, outputCount + 1`

Reply:
The buffer is mapped to have `1 + maxOutputValues` elements instead of `outputCount + 1`, so I thought we should keep this